src/ralph/trade_marks/models.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+from dj.choices import Choices, Country
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.accounts.models import Regionalizable
+from ralph.assets.country_utils import iso2_to_iso3
+from ralph.assets.models import AssetHolder, BaseObject
+from ralph.domains.models import Domain
+from ralph.lib.mixins.models import (
+    AdminAbsoluteUrlMixin,
+    NamedMixin,
+    TimeStampMixin
+)
+
+
+class TradeMarkType(Choices):
+    _ = Choices.Choice
+    word = _('Word')
+    figurative = _('Figurative')
+    word_figurative = _('Word - Figurative')
+
+
+class ProviderAdditionalMarking(
```

Comment:
What is the role of this model? I think some comment in code will be beneficial to future developers ;)

Reply:
maybe we could figure out some better name for this?

Reply:
Czy to zostało zaadresowane?

Reply:
We discussed about it out of github, and decide to leave it in this form. Whatever, I've add an comment to this class witch short explanation